renderer/src/test/onboarding.test.tsx

```diff
@@ -0,0 +1,72 @@
+import { beforeEach, describe, expect, test, vi } from 'vitest'
+import { act, fireEvent, render, screen, waitFor } from '@testing-library/react'
+
+import '../lib/station-config'
+import App from '../App'
+
+vi.mock('../lib/station-config', () => {
+  return {
+    setOnboardingCompleted: () => Promise.resolve(undefined),
+    getOnboardingCompleted: (status: boolean) => Promise.resolve(false)
+  }
+})
+
+const mockedUsedNavigate = vi.fn()
+
+vi.mock('react-router-dom', async () => {
+  const router:any = await vi.importActual('react-router-dom')
```

Comment:
It would be great if we could use the actual type of `import('react-router-dom')` instead of `any`.

Would something like this work?

```suggestion
  const router: import('react-router-dom') = await vi.importActual('react-router-dom')
```

Let's leave this for later, after this PR is landed.